messagingpush/src/main/res/xml/backup_rules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<full-backup-content>
+    <!-- Encrypted preferences should not be backed up as the key used to encrypt it will no
+    longer be present when backup is restored -->
+    <exclude
+        domain="sharedpref"
+        path="io.customer.sdk.EncryptedConfigCache.xml" />
```

Comment:
Does this hard-coded string need to match [this hard-coded string](https://github.com/customerio/customerio-android/pull/275/files#diff-c1a277cd6b2e000cf26e555a3c571509c7d81202c4e8f26d62015e2421308f4dR28)? 

If so, I suggest using a string resource value to store this string in 1 place.